src/libs/actions/Task.js

```diff
@@ -368,7 +374,7 @@ function editTaskAndNavigate(report, ownerEmail, ownerAccountID, {title, descrip
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`,
-            value: {reportName: report.reportName, description: report.description, assignee: report.managerEmail, assigneeAccountID: report.managerID},
```

Comment:
@thienlnam do you see any concerns with this? Using `managerID` to get `login` from personal details?

If you assign a task to someone you don't know.... I guess that's the main thing we need to test here

Reply:
This kiiiinda hinges on internal convo - https://expensify.slack.com/archives/C03TQ48KC/p1689252597374799

Reply:
Jack said he's get started on the new command, but idk what the progress / status is 😅

Reply:
Nah I wasn't able to get around to the new command. But yeah the problem would likely be assinging the task to someone you don't know. I've been working on those changes here though https://github.com/Expensify/Expensify/issues/294203